airflow/ui/src/queries/useTrigger.ts

```diff
@@ -73,6 +73,8 @@ export const useTrigger = ({ dagId, onSuccessConfirm }: { dagId: string; onSucce
       ? new Date(dagRunRequestBody.dataIntervalEnd)
       : undefined;
 
+    const LogicalDate = dagRunRequestBody.logicalDate ? new Date(dagRunRequestBody.logicalDate) : undefined;
```

Comment:
Oh I didn't catch this before but all of these variables should start with a lower case letter.

Do you mind fixing it here and for dataIntervalStart and dataIntervalEnd?

Reply:
ofc I'll do that. True that's weird :p I naively followed the convention, but lower case is more appropriate :)

Reply:
Done